etc/kubernetes/helm/enterprise-gateway/values.yaml

```diff
@@ -1,19 +1,49 @@
+global:
+  serviceAccountName: 'enterprise-gateway-sa'
+  # ImagePullSecrets for all ServiceAccount, list of secrets in the same namespace
+  # to use for pulling any images in pods that reference this ServiceAccount.
+  # Must be set for any cluster configured with private docker registry.
+  imagePullSecrets: []
+  # - private-registry-key
```

Comment:
Are you aware of #1053 and would you mind taking a look at it?  My experience in this area is, well, non-existent, so another view would be appreciated!

Reply:
Excellent!  If you want "extra credit" you could add a little "Private Registries" section to [this portion of the docs](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/operators/deploy-kubernetes.html#kernel-image-puller). :wink:

Reply:
working on it